Add Cash option in Protonmail payments (#1386)
privacyguides/privacyguides.org: docs/email.en.md

??? check "Private Payment Methods"

>     Proton Mail [accepts](https://proton.me/support/payment-options) Bitcoin and cash by mail, in addition to standard credit/debit card and PayPal payments.

Reviewer: A comma isn't needed when the dependent clause comes after the independent clause :p

```suggestion
    Proton Mail [accepts](https://proton.me/support/payment-options) Bitcoin and cash by mail in addition to standard credit/debit card and PayPal payments.
```

>     Proton Mail [accepts](https://proton.me/support/payment-options) Bitcoin in addition to accepting credit/debit cards and PayPal. They also accept cash by mail

Reviewer 2: ```suggestion
    Proton Mail [accepts](https://proton.me/support/payment-options) Bitcoin and cash by mail, in addition to standard credit/debit card and PayPal payments.
```
